.github/workflows/docsearch-scraper.yml

```diff
@@ -0,0 +1,20 @@
+name: Devportal Scraper
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Scraper running
+    steps:
+    - uses: actions/checkout@v2
+    - name: Devportal Scraper
+      uses: vtexdocs/devportal-docsearch-action@main
+      with:
+        algolia_application_id: 'A4TXCBOC74'
```

Comment:
What do you think about adding that as a secret too?

Reply:
Nice! It's a good idea to protect our App ID as well, which makes it easier to change the ID in the future.